src/towncrier/_builder.py

```diff
@@ -129,10 +137,19 @@ def find_fragments(
         file_content = {}
 
         for basename in files:
+            if basename in ignored_files:
```

Comment:
Does it make sense to do a case insenstive match  ?

```suggestion
            if basename.lower() in ignored_files:
```

and making sure values in `ignored_files` are kept lower case.